ee/billing/test/test_quota_limiting.py

```diff
@@ -76,25 +85,110 @@ def test_billing_rate_limit(self) -> None:
                     team=self.team,
                 )
         time.sleep(1)
-        result = update_all_org_billing_quotas()
         org_id = str(self.organization.id)
-        assert result["events"] == {org_id: 1612137599}
-        assert result["recordings"] == {}
-        assert result["rows_synced"] == {}
 
-        assert self.redis_client.zrange(f"{QUOTA_LIMITER_CACHE_KEY}events", 0, -1) == [
-            self.team.api_token.encode("UTF-8")
-        ]
-        assert self.redis_client.zrange(f"{QUOTA_LIMITER_CACHE_KEY}recordings", 0, -1) == []
-        assert self.redis_client.zrange(f"{QUOTA_LIMITER_CACHE_KEY}rows_synced", 0, -1) == []
+        with freeze_time("2021-01-25T22:09:14.252Z"):
+            # Should be data_retained until Feb 1 2021
+            quota_limited_orgs, data_retained_orgs = update_all_org_billing_quotas()
+            assert data_retained_orgs["events"] == {org_id: 1612137600}
```

Comment:
does this org_id change ever?

Reply:
This isn't the org_id but the time at which data retention for this org should end.

Reply:
It's inconsistent with the other object for quota_limited_orgs...Will think about it.

Reply:
The org_id in this key gets replaced with the organization.id used in this test so the value we're asserting against looks more like {'uuid123123123': 1612137600}

Reply:
ohh I get it 👍